### PR TITLE
LPS-170191 - [Raylife AP] Add status column in the table

### PR DESCRIPTION
**Pull request by pedroliveirav**
[LPS-170191](https://issues.liferay.com/browse/LPS-170191)

### CODE REVIEW

**Comment by pedroliveirav** [comment]
ci:test:sf

**Comment by pedroliveirav** [comment]
ci:test:sf

File: modules/apps/site-initializer/site-initializer-raylife-ap/extra/remote-app/src/routes/policies/pages/PoliciesTable/index.tsx [hunk 1]
```diff
@@ -503,8 +507,7 @@ const PoliciesTable = () => {
 						}
 						if (renewalDue === 0) {
 							return 'Due Today';
-						}
-						else {
+						} else {
```
Comment (eliasantosf) [inline modules/apps/site-initializer/site-initializer-raylife-ap/extra/remote-app/src/routes/policies/pages/PoliciesTable/index.tsx:510]: Run **afcb** to adjust these lines. If possible remove the else.

File: modules/apps/site-initializer/site-initializer-raylife-ap/extra/remote-app/src/routes/policies/pages/PoliciesTable/index.tsx [hunk 2]
```diff
@@ -327,8 +328,7 @@ const PoliciesTable = () => {
 					);
 				})
 			);
-		}
-		else {
+		} else {
```
Comment (eliasantosf) [inline modules/apps/site-initializer/site-initializer-raylife-ap/extra/remote-app/src/routes/policies/pages/PoliciesTable/index.tsx:331]: Run afcb to adjust these lines. If possible remove the else.

**Review by eliasantosf — CHANGES_REQUESTED** [review]

**Comment by pedroliveirav** [comment]
ci:test:sf

File: modules/apps/site-initializer/site-initializer-raylife-ap/extra/remote-app/src/routes/policies/pages/PoliciesTable/index.tsx (hunk 1 above)
Comment (pedroliveirav) [reply modules/apps/site-initializer/site-initializer-raylife-ap/extra/remote-app/src/routes/policies/pages/PoliciesTable/index.tsx]: Done.

File: modules/apps/site-initializer/site-initializer-raylife-ap/extra/remote-app/src/routes/policies/pages/PoliciesTable/index.tsx (hunk 2 above)
Comment (pedroliveirav) [reply modules/apps/site-initializer/site-initializer-raylife-ap/extra/remote-app/src/routes/policies/pages/PoliciesTable/index.tsx]: Done.

**Review by joanagrego — COMMENTED** [review]
Approved! Thank you @pedroliveirav :white_check_mark:

**Review by joanagrego — APPROVED** [review]

**Review by eliasantosf — APPROVED** [review]
LGTM :fire: Thanks @pedroliveirav

**Comment by silasbsi** [comment]
ci:forward

**Comment by pedroliveirav** [comment]
ci:forward

**Comment by pedroliveirav** [comment]
ci:forward

**Comment by pedroliveirav** [comment]
ci:forward

**Pull request closed without merging**